HW_1/task_1.py

```diff
@@ -0,0 +1,10 @@
+    # Задача 2: Найдите сумму цифр трехзначного числа.
+
+Number = str(input('Введите трехзначное число '))
```

Comment:
Обычно в Python имена переменных функций и так далее (bсключение составляют имена классов) начинаются с символа в нижнем регистре. Поэтому, кстати, GitHub подсвечивает вам этe переменную иначе.

Reply:
Функция ввода данных уже возвращает тип `str`.
```suggestion
Number = input('Введите трехзначное число ')
```